lib/utils/strings.ts

```diff
@@ -17,6 +17,17 @@ export const getTitleString = (
   return titleString;
 };
 
+// more sophisticated mapping can be applied
+export const getTitleStringFromPathname = (pathname: string): string => {
+  let title;
+  if (pathname === '/') {
+    title = 'Home';
+  } else {
+    title = pathname.substring(1);
```

Comment:
hm. shouldn't we rather pass path (instead of pathname) and choose the last string in the path array as the title? In any case, if we have more complicated paths, we certainly don't want our title to be 'settings/privacy' or whatever. And if we only have signin and signup, where this is relevant, why are we not more explicit and use a map instead of string manipulation? Something like 
```usePathForTitleString = (path: string[]) : string => {
if (!path.length) return 'home';
const firstComponent = path[0];
switch (firstComponent) {
 case 'signin': return 'signin';
case 'talents': 
// getTalentByID(path[1]) graphql query goes here from which we get the name
default:
return '';
}
}```

Reply:
we cannot have the graphql query there because it's not within a React Function Component.

I know that the function is very basic right now. That's why I left the comment `// more sophisticated mapping can be applied`.
Actually I would prefer to upgrade that logic in another PR. Because upgrading the title mapping was actually not the goal of this PR.
I created a ticket: https://kara-jobs.atlassian.net/browse/KM-100
Fine by you?

Reply:
for the time being, I suggest this:
```
export const getTitleStringFromPathname = (pathname: string): string => {
  const path = pathname.split('/').filter((segment) => segment !== '');
  if (path.length === 0) {
    return getTitleString('Home');
  }
  return getTitleString(path.join(' | '));
};
```